phnt/include/ntldr.h

```diff
@@ -649,7 +649,7 @@ typedef struct _PS_MITIGATION_AUDIT_OPTIONS_MAP
 typedef struct _PS_SYSTEM_DLL_INIT_BLOCK
 {
     ULONG Size;
-    ULONG_PTR SystemDllWowRelocation;
+    ULONG_PTR SystemDllWowRelocation; // current offset since REDSTONE2
```

Comment:
```
// since RS2
```

Reply:
Do we not want to differentiate between fields that are new to a specific OS version vs. fields that existed before but changed their offset?